### ISSUE TITLE
Possibility of training the model to predict pepide with modifications

### ISSUE THREAD

**Issue by irleader** [lkytal/PepNet#5]
Hi,

Our dataset have lots of modifications which are important, is it possible to modify the model a bit, such that we can train PepNet from scratch by ourselves to be able to include modifications? Otherwise we are still unable to use PepNet for our work.

Best regards

**Comment by lkytal** [comment]
Yes, and we expect to release new models that support common modifications. And if you have good training data, it can help us refine our model if you could share some of your data, thanks
